pkg/client/client.go

```diff
@@ -50,10 +51,15 @@ func (c *Client) BuildConfigs(namespace string) BuildConfigInterface {
 }
 
 // BuildLogs provides a REST client for BuildLogs
-func (c *Client) BuildLogs(namespace string) BuildLogInterface {
+func (c *Client) BuildLogs(namespace string) BuildLogsInterface {
 	return newBuildLogs(c, namespace)
 }
 
+// BuildGenerate provides a REST client for Build generation
+func (c *Client) BuildGenerate(namespace string) BuildGenerateInterface {
```

Comment:
Sorry, BuildConfigInterface.  See upstream clients for Binding, Status, etc.  Verbs on the core resource are direct, verbs on sub resources are "UpdateStatus".

----- Original Message -----
> IT should be on the Build interface.
> 
> ----- Original Message -----
> > >  	return newBuildLogs(c, namespace)
> > >  }
> > >  
> > > +// BuildGenerate provides a REST client for Build generation
> > > +func (c *Client) BuildGenerate(namespace string) BuildGenerateInterface
> > > {
> > 
> > Method on the client being "Generate" is fine, my concern was w/ calling
> > the
> > interface itself "BuildGenerateInterface" (should be
> > BuildGeneratorInterface)
> > 
> > 
> > ---
> > Reply to this email directly or view it on GitHub:
> > https://github.com/openshift/origin/pull/1275/files#r26703237
>